### DISCUSSION TITLE
Remote access to Analyst VM

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#8215] [Unsupported Versions]
Hello!

Is there a preferred setup for remote desktop access to the Analyst VM  (e.g. xRDP)? I couldn't find anything in the docs. Thanks!

Cheers, Ben

**Comment by cyberkimh** [comment]
This might be of help: https://www.itzgeek.com/how-tos/linux/centos-how-tos/install-xrdp-on-centos-7-rhel-7.html
